source/scaling.rst

```diff
@@ -0,0 +1,22 @@
+.. _operator-scale:
+
+Scale Percona Distribution for PostgreSQL on Kubernetes and OpenShift
```

Comment:
not now, but for later on we need to have Horizontal Pod Autoscaler (HPA) example here + also add more information about vertical scaling and if it is possible to integrate with Vertical Pod Autoscaler (VPA).